srcpkgs/zlib-ng/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'zlib-ng'
+pkgname=zlib-ng
+version=2.0.1
+revision=1
+build_style=cmake
+configure_args="-DZLIB_COMPAT=ON"
+short_desc="Zlib replacement with optimizations for \"next generation\" systems"
+maintainer="Artur Sinila <opensource@logarithmus.dev>"
+license="Zlib"
+homepage="https://github.com/zlib-ng/zlib-ng"
+distfiles="https://github.com/zlib-ng/zlib-ng/archive/${version}.tar.gz"
+checksum=8599893f9b78bf979c1a1d6549b730367c9186560c6879590354998cc55428cf
+alternatives="zlib:libz.so.1:/usr/lib/libz.so.1.2.11.zlib-ng"
```

Comment:
I don't like making `zlib` an alternative, especially because XBPS depends on it, so if it breaks (and it *can* break), you won't even be able to run the tool to "easily" fix it.

Reply:
@ericonr speaking about possible breakage of `zlib-ng`, if you haven't removed `zlib` after installing `zlib-ng`, you can just run `xbps-alternatives -s zlib`. Even if you've removed `zlib` there's `xbps-static`. As the last resort there's an option to `chroot` into your system via Void installation media and then copy `libz.so` from it.

Reply:
`xbps-alternatives` can't run if the symlink is broken. It would require either static XBPS or manual symlinking.